packages/derive/src/callable_point.rs

```diff
@@ -69,3 +69,61 @@ fn make_call_origin_and_return(
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use syn::{parse_quote, ItemFn};
+
+    const PART_CALL_TO_ORIGIN_ARG0: &str = "foo () ;";
+    const PART_CALL_TO_VEC: &str = "cosmwasm_std :: to_vec (";
+    const PART_CALL_TO_RELEASE_BUFFER: &str = "cosmwasm_std :: memory :: release_buffer (";
```

Comment:
`TO` is not needed, or PART_TO_XXX is correct.

"call [a function]" is the correct grammar.

Reply:
Following the suggestion below, we have changed the test style and all related PART content has been removed.